hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/DefaultServiceEntityEnricher.java

```diff
@@ -55,11 +53,11 @@ public void enrichEvent(StructuredTrace trace, Event event) {
       return;
     }
 
-    // If there is jaegerServiceName present in the span, just go ahead and create a service
+    // If there is serviceName present in the span, just go ahead and create a service
     // entity with those details. This is to support BareMetal case.
-    String jaegerService =
-        SpanAttributeUtils.getStringAttribute(event, JAEGER_SERVICE_NAME_ATTR_NAME);
-    if (jaegerService != null) {
+    String serviceName =
+        event.getServiceName();
+    if (serviceName != null) {
```

Comment:
the collector is setting this attribute now ?

Reply:
It's been set by ```JaegerSpanNormaliser```